pkg/app/piped/imagewatcher/watcher.go

```diff
@@ -114,11 +114,16 @@ func (w *watcher) run(ctx context.Context, provider imageprovider.Provider, inte
 				updates = append(updates, u...)
 			}
 			if len(updates) == 0 {
-				w.logger.Info("no image to be updated")
+				w.logger.Info("no image to be updated",
+					zap.String("image-provider", provider.Name()),
+				)
 				continue
 			}
 			if err := update(updates); err != nil {
-				w.logger.Error("failed to update image", zap.Error(err))
+				w.logger.Error("failed to update image", zap.String("image-provider",
+					provider.Name()),
```

Comment:
`zap.String("image-provider",
					provider.Name())` should be in a same line.

Reply:
Oops, thanks